style.css

```diff
@@ -0,0 +1,379 @@
+html {
+  align-items: center;
+  width: 100%;
+  display: flex;
+}
+
+body {
+  background: #e9e8e8b6;
+  position: absolute;
+  width: 100%;
+  border: 1px;
+  margin: 20px;
+  left: 0;
+  top: 0;
+  align-items: center;
+}
+
+header {
+  width: 375px;
+  height: 60px;
+  display: flex;
+  justify-content: space-between;
+  align-items: basline;
+  margin: 20px;
+  padding-left: 0;
+}
+
+header p {
+  color: #6070ff;
+  width: 140px;
+  height: 30px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 29px;
+  line-height: 20px;
+  display: flex;
+  text-align: start;
+}
+
+header i {
+  font-size: 2.2rem;
+  margin-right: 50px;
+}
+
+button a {
+  text-decoration: none;
+}
+
+.headline a {
+  text-decoration: none;
+}
+
+.wraper {
+  background-color: white;
+  width: 380px;
+  border-radius: 0 0 0 80px;
+}
+
+.headline {
+  background-image: url(/header-shapes\ mobile.png);
+  background-size: 375px;
+  background-repeat: no-repeat;
+  background-position: left + 10px;
+  background-position-y: 40px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex start;
+  padding: 114px 24px;
+  gap: 12px;
+  width: 350px;
+  height: 424px;
+  left: calc(45% - 375px / 2);
+  top: 20px;
+  border-radius: 0 0 8px 0;
+}
+
+h1 {
+  width: 300px;
+  height: 150px;
+  margin: 1px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.details {
+  width: 300px;
+  height: 140px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-size: 16px;
+  line-height: 28px;
+  color: #344563;
+  margin-bottom: 40px;
+  margin-top: -1px;
+}
+
+h4 {
+  font-family: 'Poppins', sans-serif;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  text-decoration: none;
+  font-style: normal;
+  color: #7f8cff;
+  margin: 0;
+}
+
+.social ul {
+  display: flex;
+  width: 150px;
+  height: 40px;
+  justify-content: space-between;
+  font-size: 30px;
+  list-style: none;
+  margin: 0;
+  padding: 0;
```

Comment:
As requested by the previous design kindly check and improve your design to make it look similar to the actual design. For instance in the images below, your are using different colors compared to the actual design, your design is missing the `Get my resume` button as well as the background image 


### Your design

![image](https://user-images.githubusercontent.com/65233860/220653874-b683a6dd-5c3e-4539-8e18-59c7bfc33903.png)

### Figma design

![image](https://user-images.githubusercontent.com/65233860/220653918-b6b759c6-1b04-43ec-8c7a-dc383fd9d7bf.png)